### ISSUE TITLE
Call to undefined function Gumlet\imagecreatefromjpeg() 

### ISSUE THREAD

**Issue by Script47** [gumlet/php-image-resize#168]
I'm using this library (1.9.*) in Laravel (v6) and getting the following exception:

>Call to undefined function Gumlet\imagecreatefromjpeg() 

when running the following line of code:

```php
$imgObj = ImageResize::createFromString($var);
```

The trace narrows it down to `ImageResize.php:170`, which is:

```php
$img = imagecreatefromjpeg($filename);
```

If I run `php -m` the output is:

```
$ php -m
[PHP Modules]
bz2
calendar
Core
ctype
curl
date
dom
exif
fileinfo
filter
ftp
gd
gettext
hash
iconv
json
libxml
mbstring
mysqli
mysqlnd
openssl
pcntl
pcre
PDO
pdo_mysql
pdo_sqlite
Phar
readline
Reflection
session
SimpleXML
sockets
SPL
sqlite3
standard
tokenizer
wddx
xml
xmlreader
xmlwriter
xsl
zlib

[Zend Modules]
```

The only thing I can think of is that it's trying to find that method in Gumlet\imagecreatefromjpeg() instead of global PHP namespace.

Thoughts?

**Comment by peter279k** [comment]
We can use the `$img = \imagecreatefromjpeg($filename);` and it will fix this issue.

The reason is about namespace checking on PHP runtime.

**Comment by Script47** [comment]
@peter279k 

One thing that remains unclear to me is that how come this isn't affecting everyone?

**Comment by peter279k** [comment]
Well, it's about the internal PHP version problem.

It will happen on some PHP versions and other will not. I also get this problem on some PHP versions.

**Comment by FuTuL** [comment]
It might be a little late. This usually happens if the gd is configured without jpeg support. You can view the options here (https://www.php.net/manual/en/image.installation.php). For example, in a docker you need to execute `docker-php-ext-configure gd --with-jpeg`.

**Comment by Sergeyii** [comment]
Try to use \imagecreatefromjpeg from global PHP namespace - outputs error "attempted to call function imagecreatefromjpeg from global namespace".

In my case the problem was solved in docker PHP 7.4-fpm by adding line before install:
**docker-php-ext-configure gd --with-jpeg** && \
docker-php-ext-install gd \
